spec/factories/temp_sites.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  factory :temp_site do
```

Comment:
I think `FactoryBot` can do inheritance so we could also DRY this up.

Reply:
My suggestion here isn't relevant if we're cloning the models, feel free to ignore.